manifests/s/SparkLabs/openvpn-configuration-generator/1.0.9.1009/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=NVS0.7-1-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.0.0.schema.json
+
+PackageIdentifier: SparkLabs.openvpn-configuration-generator
+PackageVersion: 1.0.9.1009
+PackageLocale: en-US
+Publisher: SparkLabs
```

```suggestion
Publisher: SparkLabs Pty Ltd
```
This must match Control Panel.